### DISCUSSION TITLE
change  issue templates

### DISCUSSION THREAD

**Discussion by lgsmith** [GrossfieldLab/loos#96] [Developers]
I'd like to change default issue template to use more markup from markdown. Instead of having the various sections of an issue report be a **boldface** title and then a sentence in plaintext explaining what should go there, I think that the title should have a section marker, or if that's too large then a subsection marker, and the instructions should be in a comment. This would look something like the following (note that I  escaped the leading comment mark so that you could see what I figured the comments ought to read like):

# Describe the bug
\

# To Reproduce
\

# Expected behavior
\

# LOOS version and platform
\

# Additional context
\

I believe  that this will make it easier for people to read the reports, because it'll make skimming a bit better (you get the cross-lines from the use of section, which I like) and it'll get rid of the prompt text that is currently co-mingled with the answers when some users fill these out. I tend to delete it when I file issues, but I've noted that others don't necessarily.

**Comment by agrossfield** [comment]
Addressed by https://github.com/GrossfieldLab/loos/commit/43b33d64baae69e7888adc649efa08ad132574a3

**Reply by agrossfield** [reply]
Finally went back and changed the prompts to comments.
